internal/databases/mongo/binary/mongod.go

```diff
@@ -38,7 +38,7 @@ func CreateMongodService(ctx context.Context, appName, mongodbURI string) (*Mong
 		func() error {
 			mongoClient, err = mongo.Connect(ctx,
 				options.Client().ApplyURI(mongodbURI).
-					SetServerSelectionTimeout(10*time.Minute).
+					SetServerSelectionTimeout(timeout).
 					SetConnectTimeout(10*time.Minute).
```

Comment:
Can we add this timeout to ConnectTimeout too?

Reply:
Yes, but I don't see the point of doing this